### PR TITLE
return type hint added

### PR DESCRIPTION
**Pull request by Gustavobflh**
### Describe your change: Added a Hubble Parameter calculator and fixed the return hint.

Closes #7671 

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by cclauss** [comment]
Please run this on your own computer:
`python3 -m doctest -v physics/hubble_parameter.py`

**Comment by Gustavobflh** [comment]
I think now ti's ok!

File: physics/hubble_parameter.py [hunk 1]
```diff
@@ -54,17 +54,20 @@ def hubble_parameter(
     result : Hubble parameter in and the unit km/s/Mpc (the unit can be
     changed if you want, just need to change the unit of the Hubble constant)
 
-    >>> hubble_parameter(hubble_constant=68.3, radiation_density=1e-4, matter_density=-0.3, dark_energy=0.7, redshift=1)
+    >>> hubble_parameter(hubble_constant=68.3, radiation_density=1e-4, \
```
Comment (cclauss) [inline physics/hubble_parameter.py:57]: PEP8 says that backslashes are a bad idea in Python code.

Comment (Gustavobflh) [reply physics/hubble_parameter.py:57]: Without it the tests were failing. I don't know exactly  how to break line without it. This was the way I found. There is another?

Comment (cclauss) [reply physics/hubble_parameter.py:57]: You can wrap lines at will between  (), {}, [],

Comment (Gustavobflh) [reply physics/hubble_parameter.py:57]: But when I'm running without the backslashes the tests are failing and I don't know why.

Comment (Gustavobflh) [reply physics/hubble_parameter.py:57]: But thank you for the tip.  One more question,  this pull request was accepted?

Comment (cclauss) [reply physics/hubble_parameter.py:57]: Three lines are > 88 characters as discussed in CONTRIBUTING.md  Under the `Details` link to the right of the ❌ ...
```
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

physics/hubble_parameter.py:57:89: E501 line too long (120 > 88 characters)
physics/hubble_parameter.py:62:89: E501 line too long (120 > 88 characters)
physics/hubble_parameter.py:67:89: E501 line too long (120 > 88 characters)
```

Comment (Gustavobflh) [reply physics/hubble_parameter.py:57]: So, but you close de pr, I can try to fix it without using the backslash and make another pr. It's ok?

Comment (Gustavobflh) [reply physics/hubble_parameter.py:57]: So, I found the solution. in doctest I have to use ... to break line. 
I have to do another pr or leave this here?

Comment (cclauss) [reply physics/hubble_parameter.py:57]: Do another PR please.

**Pull request closed without merging**